pages/api/check-status.ts

```diff
@@ -122,9 +130,9 @@ export const searchPassportStatusMock = (
   )
 
   if (applicationResponse) {
-    res.send(mapToCheckStatusApiResponse(applicationResponse))
-    return
+    logger.debug(applicationResponse)
+    return res.send(mapToCheckStatusApiResponse(applicationResponse))
   }
-
-  res.status(404).send('Passport Status Not Found')
+  logger.error(`Status 404: Passport Status Not Found`)
```

Comment:
I would log this as `debug` because this isn't an application error and we're going to have potentially millions of requests.